app/views/layouts/application.html.erb

```diff
@@ -55,7 +74,7 @@
         <div class="footer_info">
           <p>Created by the <a href="http://cdrh.unl.edu">Center for Digital Research in the Humanities</a>.</p>
           <div class="footer_logo">
-            <a href="http://www.unl.edu"><%# image_tag 'unl_black.png' %></a>
+            <a href="http://www.unl.edu"><%= image_tag 'unl_black.png' %></a>
```

Comment:
unfortunately, this is going to fail when a new app starts.  The fix for it is in a different branch:  https://github.com/CDRH/orchid/commit/6ec1cea66b0f03a20c8591b11d7c4b7a36e22a3e

I recommend that we simply leave it commented for now to avoid the app failing on starting until the other branch is merged